node-services/user-service/controllers/validateParameters.js

```diff
@@ -0,0 +1,16 @@
+const validate = {
+  validateEmail: (email) => {
```

Comment:
some libraries just throws an error when validation fails, advantage is you don't need to check boolean returned value https://guava.dev/releases/19.0/api/docs/com/google/common/base/Preconditions.html#checkNotNull(T)